matrix-sdk/src/androidTest/java/org/matrix/androidsdk/common/CommonTestHelper.java

```diff
@@ -355,4 +357,46 @@ public void onSuccess(Credentials credentials) {
     public void await(CountDownLatch latch) throws InterruptedException {
         Assert.assertTrue(latch.await(TestConstants.AWAIT_TIME_OUT_MILLIS, TimeUnit.MILLISECONDS));
     }
+
+    /**
+     * Clear all non null sessions in lazy loading scenario data
+     *
+     * @param data
+     */
+    public void clearAllSessions(LazyLoadingScenarioData data) {
```

Comment:
Don't add this to Common as it's belong to lazy loading

Reply:
done